src/Resources/config/packages/sentry.yaml

```diff
@@ -0,0 +1,8 @@
+sentry:
+  options:
+    ignore_exceptions:
+      - 'Shopware\Storefront\Framework\Routing\Exception\SalesChannelMappingException'
```

Comment:
Could be configured in the core as a shopware pull request and would automatically picked up by https://github.com/FriendsOfShopware/shopware-sentry-bundle/blob/main/src/Integration/UseShopwareExceptionIgnores.php ?

vendor/shopware/core/Framework/Resources/config/packages/framework.yaml:85